bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -213,6 +213,7 @@ impl SlidingSyncRoom {
                 .await;
         };
 
+        let mut ignore_user_list_rx = self.client.get_ignore_user_list_broadcast_tx().subscribe();
```

Comment:
I would rename this to `ignore_user_list_changed_broadcast_tx`, although I'm not sure if the `_tx` is actually necessary. Maybe we can go with `broadcast_sender` in both